gensim/test/test_sklearn_api.py

```diff
@@ -498,7 +498,6 @@ def testPersistence(self):
         original_matrix = self.model.transform(original_bow)
         passed = numpy.allclose(loaded_matrix, original_matrix, atol=1e-1)
         self.assertTrue(passed)
-
     def testModelNotFitted(self):
```

Comment:
Need to add more tests (for new functionality)

Reply:
I have that in my checklist but before that I need to pass the already present tests.